src/lib.rs

```diff
@@ -379,9 +413,7 @@ where
         self.i2c.write(
             self.address,
             &[(Oscillator::COMMAND | self.oscillator_state).bits()],
-        )?;
-
-        Ok(())
+        )
     }
```

Comment:
Removing the endings of similar functions is just a preference really, but makes no difference.